Src/DependencyCollector/Shared/Implementation/DependencyCollectorEventSource.cs

```diff
@@ -267,8 +267,12 @@ private string GetApplicationName()
             try
             {
 #if NETCORE
+#if NETSTANDARD1_3
+                name = "Undefined"; // netstandard1.3 does not have API to retrieve Assembly/AppDomain name
```

Comment:
"Undefined" does not seem like an acceptable name.  Even in 1.3 there must be something we can call.